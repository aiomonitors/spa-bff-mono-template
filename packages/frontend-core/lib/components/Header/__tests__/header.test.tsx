import React from 'react';
import { renderToString } from 'react-dom/server';
import { Header } from '..';

describe('Header Component', () => {
  it('Should match the snapshot', () => {
    const renderedString = renderToString(<Header />);
    expect(renderedString).toMatchSnapshot();
  })
})
