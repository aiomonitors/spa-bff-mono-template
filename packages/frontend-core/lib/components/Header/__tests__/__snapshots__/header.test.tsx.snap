// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should match the snapshot 1`] = `"<h1>This is a test header</h1>"`;
